documentation/templates/python/class.html

```diff
@@ -139,6 +142,16 @@ <h2><a href="#data">Data</a></h2>
           </dl>
         </section>
         {% endif %}
+        {% if page.exceptions %}
```

Comment:
I've created a separate variable for exceptions. The alternatives of `has_exceptions`/`has_classes` variables or manual checks in templates look worse to me.

Reply:
I'd actually keep them together in `page.classes` and branch in the template. That way people have more flexibility, either have them separate or  put them sorted together with classes into a single "Classes" section in a modified template if they want.